test/README.md

```diff
@@ -27,6 +27,19 @@ $ npm run test-interactive
 
 ### Developer utility
 
+macOS:
+
+```sh
+$ docker ps --all | grep test_hs
+$ docker logs -f --tail 10 matrix_public_archive_test-hs1-1
+$ docker logs -f --tail 10 matrix_public_archive_test-hs2-1
+
+$ docker stop matrix_public_archive_test-hs1-1 matrix_public_archive_test-hs2-1
+$ docker rm matrix_public_archive_test-hs1-1 matrix_public_archive_test-hs2-1
+```
```

Comment:
Is this actually a macOS vs Windows Docker difference? Or is it just a Docker version difference?

Reply:
On macOS, just had it create the other numbering of this. idk

Reply:
Removed until I see it again.